js/main.js

```diff
@@ -97,32 +98,40 @@ window.addEventListener('DOMContentLoaded', appendRenderEntry);
 // listen for a click event on the entries tab
 // if user presses entries tab, remove the hidden class from the data view entries element
 // add the hidden class to the form
-$entriesTab.addEventListener('click', clickEntries);
-function clickEntries(event) {
-  data.view = 'entries';
-  $entriesPage.setAttribute('class', '$entriesPage');
-  $entryForm.setAttribute('class', '$entryForm hidden');
-} if (data.entries.length > 0) {
-  $noEntry.setAttribute('class', 'no-entries hidden');
-} else {
-  $noEntry.setAttribute('class', 'no-entries');
+$entriesTab.addEventListener('click', handleViewSwitch);
+$newEntry.addEventListener('click', handleViewSwitch);
+
+function handleViewSwitch(event) {
+  var viewName = event.target.getAttribute('data-view');
+  switchViews(viewName);
+}
+
+function switchViews(string) {
+  for (var i = 0; i < $dataView.length; i++) {
+    if ($dataView[i].getAttribute('data-view') !== string) {
+      $dataView[i].classList.add('hidden');
+    } else {
+      $dataView[i].classList.remove('hidden');
+      data.view = string;
+    }
+    if (data.entries.length > 0) {
+      $noEntry.setAttribute('class', 'no-entries hidden');
+    } else {
+      $noEntry.setAttribute('class', 'no-entries');
+    }
+  }
 }
+
 // add an event listener to the new button
 // if user presses the button, remove hidden class from the form
 // add hidden class to the data view entries
-$newEntry.addEventListener('click', newEntry);
-function newEntry(event) {
-  data.view = 'entry-form';
-  $entryForm.setAttribute('class', '$entryForm');
-  $entriesPage.setAttribute('class', '$entriesPage hidden');
-}
 
 // if the window = entry form
 // load the page with the blank form
 // else if window = entries page
 // load the page with the entries
 if (data.view === 'entry-form') {
-  $submitForm();
+  switchViews(data.view);
 } else if (data.view === 'entries') {
-  clickEntries();
+  switchViews(data.view);
 }
```

Comment:
Both blocks of this conditional are doing the same thing, so the conditional is unnecessary. You can just call `switchView`